### PR TITLE
Resolve name conflicts

### PR DESCRIPTION
**Pull request by markpokornycos**
## Summary
Adds LD as a prefix to EventSource identifiers, project, target, and directories. These changes should reduce name conflicts with other items, particularly with other imports of the base EventSource library

## How to test
1. Build the framework for all devices

- [n/a] Adds unit tests as appropriate
- [X] Works on updated sdk
- [n/a] Works on a real device

## Areas of the sdk affected
- Events

## Changelog
- Updated project settings to remove build warnings

### CODE REVIEW

**Comment by markpokornycos** [comment]
CircleCI complains that there is no test Target for LDEventSource. This is consistent with the original EventSource library (see build 1) and should not prevent this PR from being accepted

File: LDEventSource/Info.plist [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.3.2</string>
+	<string>3.0.0</string>
```
Comment (arun251) [inline LDEventSource/Info.plist:18]: We're not so good about updating this. I'll add it to the release checklist, but is there a way to refactor the setup so we don't repeat the version string in multiple places?

File: LDEventSource/LDEventSource.m [hunk 2]
```diff
@@ -72,7 +72,7 @@ - (instancetype)initWithURL:(NSURL *)URL httpHeaders:(NSDictionary<NSString*, NS
         _retryInterval = ES_RETRY_INTERVAL;
         _retryAttempt = 0;
         _httpRequestHeaders = headers;
-        messageQueue = dispatch_queue_create("co.cwbrn.eventsource-queue", DISPATCH_QUEUE_SERIAL);
+        messageQueue = dispatch_queue_create("co.cwbrn.ldeventsource-queue", DISPATCH_QUEUE_SERIAL);
```
Comment (arun251) [inline LDEventSource/LDEventSource.m]: If we're changing this, it might make sense to change the prefix to `com.launchdarkly`.

**Review by arun251 — APPROVED** [review]

**Review by stephengazzard — APPROVED** [review]
Still works well. Can we turn CircleCI off for this repo, or add a test target? Not sure if we're gaining value having it fail on every PR...

**Pull request merged**